core/modules/node/node.actions.inc

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * @file
+ * Action callbacks for Node module.
+ */
+
+/**
+ * Sets the status of a node to 1 (published).
+ *
+ * @param Node $node
+ *   A node entity.
+ * @param $context
+ *   Contextual information about the triggered action.
+ *
+ * @ingroup actions
+ */
+function node_publish_action(Node $node, &$context) {
+    $node->status = NODE_PUBLISHED;
+    $node->save();
+  }
+
+
```

Comment:
Three spaces should be one.

Reply:
@docwilmot my bad, just fixed this, thanks! Not sure why the automated tests just failed after that fix because deleting those two extra line breaks was the only change I made, let me know what I should do.

Reply:
The sandboxes do strange things sometimes.